bakery/src/tests/test_cli.js

```diff
@@ -196,6 +196,7 @@ test('stable flow in pdf and distribution pipeline', async t => {
     ])
     const buildPdfResult = await completion(buildPdf)
     t.truthy(fs.existsSync(`${outputDir}/${bookId}/artifacts/collection.pdf`), formatSubprocessOutput(buildPdfResult))
+    t.is(fs.readFileSync(`${outputDir}/${bookId}/artifacts/pdf_url`, { encoding: 'utf8' }), 'https://none.s3.amazonaws.com/collection.pdf', formatSubprocessOutput(buildPdfResult))
```

Comment:
I may well be confused at this point, but is this the expected filename? (the test passes,  know, I'm just wondering if it shouldn't be checking for something of the form `-version-server_shortname-job_id.pdf`)

Reply:
there are limits to what this can test, because the lookup step does not actually run. It's mainly ensuring that nothing erroneous is added beyond the given filename. It would have at least caught the extra ')' that we were seeing.

Reply:
Ah, that's right -- The CLI is going to always give it `collection.pdf` as `pdf_filename`. 

Couldn't think of a way to give better coverage given the limitations.